File: lesson5/c_c++/homework/homework5_3.c
//: lesson5/c_c++/homework/homework5_3.c
// Напишите процедуру, выводит на экран в столбик все цифры переданного ей
// числа, начиная с первой.
#include <stdio.h>

void numbRes(int numb_i) {
    int numb = 1;
    while (numb <= numb_i) {
        printf("numb res -> %d\n", numb);
        numb++;
    }
}

int main() {
    int numb_x;
    printf("Введите число -> ");
    scanf("%d", &numb_x);
    numbRes(numb_x);
    return 0;
}

File: lesson5/python3/homework/homework5_3.py
#!/usr/bin/python3
#: lesson5/python3/homework/homework5_3.py


def numb_res(numb):
    numb_i = 1
    while numb_i <= numb:
        print("numb res ->", numb_i)
        numb_i += 1


numb_x = int(input("Введите число -> "))
numb_res(numb_x)

